tasks/libs/common/gomodules.py

```diff
@@ -151,6 +151,7 @@ class GoModule:
     lint_targets: list[str] | None = None
     # Whether the module is an otel dependency or not
     used_by_otel: bool = False
+    legacy_go_mod_version: bool | None = None
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What about calling it more explicitly `agent_6_module`? Worth adding a description comment